### DISCUSSION TITLE
mobile 디렉토리 상태관리

### DISCUSSION THREAD

**Discussion by thdwlsgus0** [CMI-OSS/cbnu-alrami#191] [Q&A]
@CMI-OSS/cbnu_alrami_fe 
혹시 mobile쪽 디렉토리 상태관리도 redux-toolkit으로 진행하나요? ㅎㅎ

**Comment by jaryapp** [comment]
디렉터리 상태관리가 혹새 어떤걸 의미하는걸까요?
전역 상태관리?

**Reply by thdwlsgus0** [reply]
@jaryapp 맞습니다!
전역 상태관리를 말하는 거였어영

**Comment by kingyong9169** [comment]
@thdwlsgus0 혹시 다른 라이브러리 생각해보신거 있으신가요?

**Reply by thdwlsgus0** [reply]
@kingyong9169 
음 redux-toolkit을 사용하는게 기존 redux의 복잡한 환경 세팅을 덜어주는 문제를 해결하는데 도움이 된다고 생각이 들긴해서 사용하면 좋을 것 같아요.

recoil을 사용해보고 싶기도 했는데 recoil은 react에서만 사용가능한 걸로 알고 있고, 저희가 충림이 이외에 react뿐만 아니라 다른 라이브러리 혹은 프레임워크에서도 작업할 경우도 있으니까 그럴 경우를 대비해 이번 기회에 redux-toolkit을 사용해도 좋을 것 같아요~

**Reply by kingyong9169** [reply]
오 recoil 개인적으로 궁금하지만 스캇 말씀대로 RTK를 사용하는게 저는 좋을 것 같아요!

**Reply by jaryapp** [reply]
넵 저희 헌재 기술스펙상으로는 전역상태는 RTK 외부 API요청은 RTK query를 사용하려고 합니다~

**Comment by nohgijin** [comment]
이거는 현재 개발에 반영된 디스커션이여서 클로즈할게요!
